packages/email-components/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@lmc-eu/jobs-email-components",
+  "version": "1.0.0",
+  "description": "Email components for mjml jobs email templates.",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public",
+    "directory": "dist"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lmc-eu/jobs-design-system.git",
+    "directory": "packages/email-components"
+  },
+  "source": "src/index.ts",
+  "main": "lib/index.js",
+  "scripts": {
+    "css:compile": "sass --load-path=../design-tokens/src/scss src:dist",
+    "css:watch": "yarn css:compile --watch",
+    "es:lint": "eslint ./",
+    "es:lint:fix": "yarn es:lint --fix",
+    "format": "yarn format:check",
+    "format:check": "prettier --check ./",
+    "format:fix": "prettier --write ./",
+    "format:fix:changelog": "prettier --write ./CHANGELOG.md",
+    "lint:styles:default": "stylelint './**/*.scss'",
+    "postinstall": "yarn build",
+    "register-components": "node registerComponents.js",
+    "build": "yarn build:scripts",
+    "build:develop": "yarn build:scripts && yarn css:compile",
+    "build:scripts": "tsc",
+    "watch": "yarn css:watch & tsc --watch",
+    "prebuild:template": "yarn build:develop",
+    "build:template": "mjml index.mjml -o index.html && mjml ./src/templates/**.mjml -o ./html-output/",
+    "watch:template": "mjml --watch index.mjml -o index.html && mjml ./src/templates/**.mjml -o ./html-output/"
+  },
+  "devDependencies": {
```

Comment:
Please re-check your dev and production dependencies. I think you are mixing them.

Reply:
Thanks for reminding me of this. I think that now, there are only required production dependencies.